charter/index.html

```diff
@@ -392,7 +404,9 @@ <h2>Success Criteria</h2>
     <p>To promote interoperability, the group will document testing policies, processes, and procedures, taking <a href="https://solidproject.org/ED/qa">Solid QA</a> as an input.</p>
 
     <!-- Horizontal review -->
-    <p>Each specification should contain sections detailing all known security and privacy implications for implementers, web authors, and end users.</p>
+    <p>Each specification should contain sections detailing all known security and privacy implications for implementers, web authors, and end users.
+      Furthermore, the Working Group must produce a general Threat Model for the ecosystem of Linked Web Storage implementations.
+    </p>
```

Comment:
For the Review, it will be one specific (to extract Assumptions and Requirements)